gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -146,11 +146,15 @@ def add_arguments(self, parser):
              ).format(sample_info_table_schema_generator.TABLE_SUFFIX))
 
     parser.add_argument(
-        '--samples_span_multiple_files',
-        type='bool', default=True, nargs='?', const=True,
-        help=('If True sample_id will be the hash of [sample_name] thus it '
-              'will be independent of file_path, otherwise hash of '
-              '[file_path, sample_name] will be used as sample_id. '))
+        '--sample_name_encoding',
+        default=vcf_parser.SampleNameEncoding.WITHOUT_FILE_PATH.name,
+        choices=[encoding.name for encoding in vcf_parser.SampleNameEncoding],
+        help=('Choose the way sample ID should be hashed. if `{}` is supplied, '
+              'sample_id will be [sample_name] (default); alternatively, if '
```

Comment:
sample_id will be **hash value** of [sample_name] (default)

Reply:
Done.